### ISSUE TITLE
Automatically mount binfmt_misc if necessary

### ISSUE THREAD

**Issue by ollieparanoid** [postmarketOS/pmbootstrap#1135]
When running `pmbootstrap` on Alpine, you need to do:
```shell
sudo modprobe binfmt_misc
sudo mount -t binfmt_misc none /proc/sys/fs/binfmt_misc
```

... or you will get:
```
sh: can't create /proc/sys/fs/binfmt_misc/register: nonexistent directory
```

We could check if `/proc/sys/fs/binfmt_misc/register` exists, and if it does not, perform the two commands above.

See also https://wiki.postmarketos.org/wiki/Troubleshooting#sh:_can.27t_create_.2Fproc.2Fsys.2Ffs.2Fbinfmt_misc.2Fregister:_nonexistent_directory

**Comment by djames1** [comment]
I just ran into this issue flashing my Nexus 4 (lg-mako). I've never had to use that kernel module before so I had no clue what this error meant. I think displaying an error informing the user to do this would be fine. Another potential possibility might be asking the user if they want to mount it.

I think automatically loading a kernel module and mounting a directory without asking is a bad idea however.

**Comment by ollieparanoid** [comment]
A check for this was implemented in #1236, it prints out a error message pointing to the wiki (where it is explained how to mount `bintfmt_misc`. I think that is good enough, but if someone really wants to ask the user whether that should be mounted, we could reopen this issue.
